services/notification.service.js

```diff
@@ -0,0 +1,43 @@
+const UserModel = require('../models/notification.schema');
+
+const getNotification = async (notificationId) => {
```

Comment:
probably don't need most of these methods. we need 1) create notification, 2) delete notification by id, 3) get notifications for user